src/lib/utils.js

```diff
@@ -0,0 +1,5 @@
+const transformUserInput = (item) => {
+  return item.replace(/[^a-zA-Z ]/g, '').replace(/\s\s+/g, ' ');
+};
```

Comment:
This is a very nice touch, but what if I want to buy "M&M's" or "100% natural juice" :slightly_smiling_face: ?
Also, this kind of arrow functions (they can be called one-liners) can also be written like this:
```
const transformUserInput = item => item.replace(/[^a-zA-Z ]/g, '').replace(/\s\s+/g, ' ')
```
They are both the exact same thing, so you can use whatever is more readable for you :slightly_smiling_face:

Reply:
We thought removing symbols like ``'%&`` was a requirement, too ?